uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/api/v3/LeaveRequestTest.php

```diff
@@ -4366,6 +4367,61 @@ public function testTheTimeForFromAndToDateOfLeaveRequestIsAddedCorrectlyWhenLea
     $this->assertEquals($toDate->format('Y-m-d') ." 23:59:00", $leaveRequest->to_date);
   }
 
+  public function testTheTimeForFromAndToDateOfLeaveRequestStaysUntouchedWhenRequestIsTOIL() {
+    $absenceType = AbsenceTypeFabricator::fabricate([
+      'allow_accruals_request'   => true,
+      'max_leave_accrual'        => 10,
+      // The calculation unit is in days since otherwise a check for
+      // calculation unit in days will take action and this test will be invalid
```

Comment:
Not sure I understand this comment, if we don't set the calculation unit the fabricator will set it to days by default so no need to pass the calculation unit here.

Reply:
Done.